packages/core/src/telemetry/detached-flush.test.ts

```diff
@@ -27,7 +27,10 @@ test("detached flush script should run without error", async ({ common }) => {
     );
   });
 
-  rmSync(telemetryEventsFilePath);
+  // if the script ran successfully, the temporary file should be deleted
+  if (existsSync(telemetryEventsFilePath)) {
+    rmSync(telemetryEventsFilePath);
+  }
```

Comment:
I think this makes more sense as an assertion

```ts
expect(existsSync(telemetryEventsFilePath)).toBeFalsy()
```

Or something like that. If the script is leaving around dead files, that should be a test failure.

Reply:
great call